src/BuildingBlocks/EventBus/EventBusServiceBus/EventBusServiceBus.cs

```diff
@@ -148,8 +151,9 @@ private Task ExceptionReceivedHandler(ExceptionReceivedEventArgs exceptionReceiv
             return Task.CompletedTask;
         }
 
-        private async Task ProcessEvent(string eventName, string message)
+        private async Task<bool> ProcessEvent(string eventName, string message)
         {
+            var processed = false;
             if (_subsManager.HasSubscriptionsForEvent(eventName))
             {
                 using (var scope = _autofac.BeginLifetimeScope(AUTOFAC_SCOPE_NAME))
```

Comment:
What about using try/catch here and do not create a new variable?

Reply:
Yep, you could.
I just created the variable to improve readability.

Reply:
Yeah, but I talk about try/catch